proto/server/datastore/datastore.proto

```diff
@@ -363,8 +363,11 @@ service DataStore {
 
     // Retrieves all the  registered entry with the same ParentID
     rpc ListParentIDEntries(ListParentIDEntriesRequest) returns (ListParentIDEntriesResponse);
-    // Retrieves all the  registered entry with the same Selector
+    // Retrieves all the  registered entry matching exactlly the compound Selector
```

Comment:
nit: double space, extra `l` in `exactly`

Reply:
Will fix 💪